server/models/index.js

```diff
@@ -28,6 +28,10 @@ Object.keys(db).forEach(modelName => {
   }
 });
 
+if (env === "production" || env === "PRODUCTION") {
```

Comment:
What is this production and super production. Also does this mean you don't sync on dev and test

Reply:
Resolved